SparseGrids/tsgCoreOneDimensional.hpp

```diff
@@ -98,7 +98,14 @@ class CustomTabulated{
     int getQExact(int level) const{ checkLevel(level, "q-exactness"); return precision[level]; }
 
     //! \brief Get the points \b x and quadrature weights \b w associated with the rule at the \b level.
-    void getWeightsNodes(int level, std::vector<double> &w, std::vector<double> &x) const;
+    void getWeightsNodes(int level, std::vector<double> &w, std::vector<double> &x) const {
+        w.resize(getNumPoints(level));
+        x.resize(getNumPoints(level));
+        getWeightsNodes(level, w.data(), x.data());
```

Comment:
No need to do that. Usually the vector method does this, but you can leave it as the `=` version that we had before. Basically, don't change this overload, just the other one.

Reply:
Done.